File: c/3-loops/practice/lp-1.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    if (n < 0 || n > 1000) {
        printf("Invalid input\n");
    } else {
        
    
        for (int i = 1; i * i <= n; i++) {
            printf("%d ", i * i);
        }
        printf("\n");
    }

    return 0;
}

File: python/7-lists/practice/lp-1.py
try:
    a = list(map(int, input().split()))
    if any(i < 0 for i in a):
        print("Invalid input")
    else:
        result = [sum(int(digit) for digit in str(num)) for num in a]
        print(' '.join(map(str, result)))
except:
    print("Invalid input")